features/polyExplorer/src/components/discoverCard/discoverCard.jsx

```diff
@@ -3,10 +3,18 @@ import LinkButton from "../buttons/linkButton/linkButton.jsx";
 import i18n from "../../i18n";
 
 import "./discoverCard.css";
+import { I18nSection } from "@polypoly-eu/silly-i18n";
 
-const i18nHeader = "clusterStoriesPreview";
+const i18nP = new I18nSection(i18n, "clusterStoriesPreview");
 const DiscoverCard = ({ story }) => {
     return (
+<<<<<<< HEAD
```

Comment:
I think thats a merge to do here. I guess you have seen it :)

Reply:
Right, forgot to commit. Thanks anyway!